autoload/fzf/vim.vim

```diff
@@ -604,7 +604,8 @@ endfunction
 " ------------------------------------------------------------------
 
 function! s:get_git_root()
-  let root = split(system('git rev-parse --show-toplevel'), '\n')[0]
+  let bufdir = substitute(split(expand('%:p:h'), '[/\\]\.git\([/\\]\|$\)')[0], "^fugitive://", "", "")
```

Comment:
A minor style suggestion
```suggestion
  let bufdir = substitute(split(expand('%:p:h'), '[/\\]\.git\([/\\]\|$\)')[0], '^fugitive://', '', '')
```

Reply:
Fixed by 88db291 and rebased :)

Reply:
Thanks. In case you didn't know, you can just press the button and apply the suggested change to the branch.

![image](https://user-images.githubusercontent.com/700826/182618210-e76b3f83-45c1-4127-bd0c-6c4322e458c9.png)